Base/usr/share/man/man6/Solitaire.md

```diff
@@ -0,0 +1,58 @@
+## Name
+
+![Icon](/res/icons/16x16/app-solitaire.png) Solitaire - The Solitaire card game
+
+[Open](file:///bin/Solitaire)
+
+## Synopsis
+
+```**sh
+$ Solitaire
+```
+
+## Description
+
+Solitaire is an implementation of the classic solitaire card game Klondike, which became very popular in digital form during the 90s after being included in Microsoft Windows OS.
+
+The game is played with a standard 52-card deck. Objective of the game is completing four distinct ordered sequences of cards (*foundation*), one per suit, from Ace to King. 
+
+After shuffling the deck, the game starts, with a setup (*tableau*) of seven vertical piles of cards, increasingly longer: the first pile is one card long, the seventh is seven cards long. Only the bottom card of each pile is upturned and accessible. The unused cards (*stock*) are kept aside, covered. 
```

Comment:
It's confusing because on a screen this card appears closest to the bottom of the screen, but I think it's technically the "top" card, because it's physically at the top if they were piled up. I tried to find a clear answer on this and [this](https://en.wikipedia.org/wiki/Glossary_of_card_game_terms#upcard) is the only description I found and agrees with it being the top card.

```suggestion
After shuffling the deck, the game starts, with a setup (*tableau*) of seven vertical piles of cards, increasingly longer: the first pile is one card long, the seventh is seven cards long. Only the top card of each pile is upturned and accessible. The unused cards (*stock*) are kept aside, covered. 
```